API/src/microservices/Assessment-Service/index.js

```diff
@@ -3,11 +3,29 @@ const { Assessments } = require(`../Database`);
 exports.submit = (assessment) => {
   return new Promise(async (resolve, reject) => { //eslint-disable-line
     try {
-      //use the bookshelf model Assessments from API/src/microservices/Database to save the assessment data in the PostgreSQL database
+      const Date = new Date();
+      const DateCreated = `${date.getFullYear()} - ${date.getMonth() + 1} - ${date.getDate()} ${date.getHours()}:${date.getMinutes()}:${date.getSeconds()}`;
+
+      var answers = { 
```

Comment:
```suggestion
      const answers = { 
```
In modern javascript, you shouldn't use "var".  Javascript supports alternative variable declaration syntax including "const" & "let".  let is for variables that might change.  const is for constants.  I recommend using those instead of var